app/serializers/comment_serializer.rb

```diff
@@ -0,0 +1,18 @@
+class CommentSerializer
+  def initialize(comment)
+    @comment = comment
+  end
+
+  def to_h
+    {
+      content: comment.content,
+      user_image_url: comment.user.image_url,
```

Comment:
maybe you could delegate from model instead of chaining in serializer ? (law of demeter and stuff)